cmd/nodecmd/sync.go

```diff
@@ -103,7 +101,10 @@ func syncSubnet(_ *cobra.Command, args []string) error {
 		return fmt.Errorf("node(s) %s failed to sync with subnet %s", untrackedNodes, subnetName)
 	}
 	ux.Logger.PrintToUser("Node(s) successfully started syncing with Subnet!")
-	ux.Logger.PrintToUser(fmt.Sprintf("Check node subnet syncing status with avalanche node status %s --subnet %s", clusterName, subnetName))
+	blockchainID := sc.Networks[network.Kind.String()].BlockchainID
```

Comment:
we need to check if `sc.Networks[network.Kind.String()]` is not nil before trying to access `BlockchainID` as it can crash